### PR TITLE
if build_image is used, it is also passed to worker build

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
Looks great! Just needs some unit testing.

**Comment by vrutkovs** [comment]
Looks good, just need tests

File: tests/plugins/test_orchestrate_build.py [hunk 1]
```diff
@@ -183,9 +188,11 @@ def test_orchestrate_build(tmpdir):
                 'platforms': ['x86_64'],
                 'build_kwargs': make_worker_build_kwargs(),
                 'osbs_client_config': str(tmpdir),
+                'worker_build_image': worker_build_image
```
Comment (lcarva) [inline tests/plugins/test_orchestrate_build.py]: Maybe only set `worker_build_image` in `args` if it's not `None`? That way you can verify that it default to `None` when not provided.

**Comment by rcerven** [comment]
build_image is in config, so create_worker_build doesn't know about it yet, as it is parsed during creating build_reqeust in set_params

**Comment by twaugh** [comment]
Checking the `build_image` used by the `Configuration` object (or used to instantiate it) would be a good test.

File: tests/plugins/test_orchestrate_build.py [hunk 2]
```diff
@@ -170,22 +171,43 @@ def make_worker_build_kwargs(**overrides):
...
-                'platforms': ['x86_64'],
-                'build_kwargs': make_worker_build_kwargs(),
-                'osbs_client_config': str(tmpdir),
-            }
+            'args': plugin_args
         }]
     )
+
+    plugin = runner.create_instance_from_plugin(OrchestrateBuildPlugin, plugin_args)
+    assert plugin.worker_build_image == worker_build_image
```
Comment (twaugh) [inline tests/plugins/test_orchestrate_build.py]: This line is testing an aspect of the implementation, which we no longer need to do.

Comment (rcerven) [inline tests/plugins/test_orchestrate_build.py]: removed

**Pull request merged**